### PR TITLE
[RPS-263] Tests for search weights

### PR DESCRIPTION
**Pull request by jacsamell**
Added tests for search weights. Also changed the boost values after
discussion with martin about the desired ordering.

Moved some tests that use the CMS to CMS folder to prevent spurious
errors.

### CODE REVIEW

File: acceptance-tests/src/test/resources/features/site/search.feature [hunk 1]
```diff
@@ -81,3 +81,17 @@ Feature: Basic search
         Given I navigate to the "home" page
         And I search for "_Attachment_Search_Term_"
         Then I should see 0 search results
+
+    Scenario: Search boosts are applied correctly to the different fields
+        Given I navigate to the "home" page
+        And I search for "WeightSearchTerm"
+            # Search results should go Series -> Publication -> Dataset
+            # within those groups the order should be Title -> Summary (-> Key Facts)
+        Then I should see search results:
```
Comment (wojtek-oledzki) [inline acceptance-tests/src/test/resources/features/site/search.feature]: maybe: "I should see search results starting with"

File: acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/pages/site/ps/SearchPage.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
+package uk.nhs.digital.ps.test.acceptance.pages.site.ps;
```
Comment (wojtek-oledzki) [inline acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/pages/site/ps/SearchPage.java]: move to "common" package instead of "ps"

Comment (jacsamell) [reply acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/pages/site/ps/SearchPage.java]: Good idea

File: acceptance-tests/src/test/resources/features/site/search.feature (hunk 1 above)
Comment (jacsamell) [reply acceptance-tests/src/test/resources/features/site/search.feature]: Good shout, will update

File: acceptance-tests/src/test/resources/features/site/search.feature [hunk 3]
```diff
@@ -81,3 +81,17 @@ Feature: Basic search
         Given I navigate to the "home" page
         And I search for "_Attachment_Search_Term_"
         Then I should see 0 search results
+
+    Scenario: Search boosts are applied correctly to the different fields
+        Given I navigate to the "home" page
+        And I search for "WeightSearchTerm"
+            # Search results should go Series -> Publication -> Dataset
+            # within those groups the order should be Title -> Summary (-> Key Facts)
+        Then I should see search results starting with:
```
Comment (ngta1) [inline acceptance-tests/src/test/resources/features/site/search.feature]: Nice and clear 👍

**Pull request merged**